### PR TITLE
Allow input expressions

### LINKED ISSUE #70: Draft 3 inputs should allow expressions
Looks like a mistake in the "initial draft 3 release" change. Draft 3 inputs definitely *should* be allowed to have default assignments, eg
```wdl
input {
  Int i = 5
}
```

### PR DESCRIPTION
**Pull request by aednichols**
Should fix #70. I _think_ the parsing tests are the right way to check this.

### CODE REVIEW

File: src/winstanley/wdl.bnf [hunk 1]
```diff
@@ -32,7 +32,11 @@ workflow_block ::= WORKFLOW WORKFLOW_IDENTIFIER_DECL LBRACE wf_body_element* RBR
 wf_body_element ::= call_block | input_block | declaration | while_loop | if_stmt | scatter_block | wf_outputs
 
 input_block ::= INPUT LBRACE input_declaration* RBRACE
-input_declaration ::= type_e IDENTIFIER {mixin="winstanley.psi.impl.WdlNamedElementImpl" implements="winstanley.psi.WdlNamedElement" methods=[getName getNameIdentifier setName]}
+input_declaration ::= type_e IDENTIFIER (EQUAL expression)? {
```
Comment (cjllanwarne) [inline src/winstanley/wdl.bnf]: This actually might now be the same as the `declaration` below. I **think** we probably should consolidate.

**Review by cjllanwarne — APPROVED** [review]

File: src/winstanley/wdl.bnf (hunk 1 above)
Comment (aednichols) [reply src/winstanley/wdl.bnf]: Excellent point, I consolidated and everything works, so let's go with that

Comment (aednichols) [reply src/winstanley/wdl.bnf]: Scratch that, one part stopped compiling in a way I don't quite understand. Going to dig some more.

**Review by cjllanwarne — APPROVED** [review]
Re-👍

**Pull request merged**